### DISCUSSION TITLE
Generate a product of Stiefel manifolds with arbitrary number

### DISCUSSION THREAD

**Discussion by yuwenchen95** [JuliaManifolds/Manifolds.jl#536] [Q&A] [answered]
Hi. I want to generate a product of Stiefel manifolds of the same dimension and the number of Stiefel manifolds can be set arbitrarily to `n`. I think it should use `ProductManifold(M_1, M_2, ..., M_n)` but have no idea how to set it with an iterator. It's too nasty to input it manually for each Stiefel manifold if n is quite large, i.e. n > 100.

**Comment by mateuszbaran** [comment]
If you need a product of the same manifold, you can use `PowerManifold`, for example `PowerManifold(Stiefel(5, 2), 100)`. There are a few different representations you can use to represent points on such power manifold: https://juliamanifolds.github.io/ManifoldsBase.jl/stable/manifolds.html#sec-power-manifold

**Comment by kellertuer** [comment, marked as answer]
To phrase Mateusz idea with a concrete example, if your Stiefel manifold is `M=Stiefel(k1,k2)` and have your large `n` of copies take

```
M2 = Powermanifold(M,n) 
```

or even shorter `M2 = M^n`.
